tests/nlu/conftest.py

```diff
@@ -36,26 +28,85 @@ def spacy_nlp_component(component_builder, default_config):
 
 
 @pytest.fixture(scope="session")
-def ner_crf_pos_feature_config():
-    return {
-        "features": [
-            ["low", "title", "upper", "pos", "pos2"],
-            ["low", "suffix3", "suffix2", "upper", "title", "digit", "pos", "pos2"],
-            ["low", "title", "upper", "pos", "pos2"],
-        ],
-        EPOCHS: 100,
-        RANDOM_SEED: 2020,
-    }
-
-
-@pytest.fixture(scope="session")
-def mitie_feature_extractor(component_builder, default_config):
+def mitie_feature_extractor(
+    component_builder: ComponentBuilder, default_config: RasaNLUModelConfig
+):
     mitie_nlp_config = {"name": "MitieNLP"}
     return component_builder.create_component(
         mitie_nlp_config, default_config
     ).extractor
 
 
+@pytest.fixture()
+def default_config() -> RasaNLUModelConfig:
+    return RasaNLUModelConfig({"language": "en", "pipeline": []})
+
+
 @pytest.fixture(scope="session")
-def default_config():
-    return config.load(CONFIG_DEFAULTS_PATH)
+def config_path() -> Text:
+    return write_file_config(
+        {
+            "language": "en",
+            "pipeline": [
+                {"name": "WhitespaceTokenizer"},
+                {"name": "CRFEntityExtractor", EPOCHS: 2},
```

Comment:
Can we fix random seeds as well in all of these, since we train and test for prediction, the corresponding tests should be deterministic.